src/community/jdbcstore/src/main/resources/jdbcstore.properties

```diff
@@ -44,4 +44,4 @@ pool.validationQuery=SELECT now()
 deleteDestinationOnRename=true
 
 # leave the following directories on the filesystem -- no spaces!
-ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp
+ignoreDirs=data,jdbcstore,workspaces,layergroups,temp,tmp,logs
```

Comment:
A quick test shows that jdbcstore does ingest the <style>.xml config files. Newly created styles do not have a <style>.xml file on-disk.

With jdbcconfig installed an enabled (before enabling jdbcstore), both jdbcstore and jdbcconfig contain a copy of the <style>.xml for all the starting styles. Actuall changes seem to go through jdbcconfig consistently, so jdbcstore is holding invalid data (nothing is using this data, but it is still being stored). If you rename a style, this can result in having two differently named styles pointing to the same style file (such that edits to one are reflected in the other). This seems problematic.